chain/src/note_source.rs

```diff
@@ -9,7 +9,6 @@ pub enum NoteSource {
     Unknown,
     Genesis,
     FundingStreamReward { epoch_index: u64 },
-    ProposalDepositRefund { proposal_id: u64 },
```

Comment:
Because the chain no longer mints refunds, this case can be removed from the `NoteSource` enum.